### ISSUE TITLE
Interactive mode is not yet supported on Windows

### ISSUE THREAD

**Issue by psfpro** [docker/for-win#302]
### Actual behavior
When I run comand `docker-compose run php composer install`
I got error
```
Interactive mode is not yet supported on Windows.
Please pass the -d flag when using `docker-compose run`.
```

### Information
A diagnostic was uploaded with id: 99ACFB01-68DE-4FBC-8BF5-ABF179D37CEE/2016-12-09_09-56-35

**Comment by rneugeba** [comment]
This is a known limitation in compose 1.8 which is fixed in 1.9 (see https://github.com/docker/compose/issues/3194).

We plan to update compose in the stable branch to 1.9 soon, but already ship compose 1.9 in the beta channel, so I'm closing this issue.

Please try the beta channel from [here](https://docs.docker.com/docker-for-windows/) and re-open if it doesn't fix your issue.

**Comment by psfpro** [comment]
Thank you!
